src/components/panel/demoGroups/index.html

```diff
@@ -0,0 +1,44 @@
+<div ng-controller="PanelGroupsCtrl as ctrl" ng-cloak>
+
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <md-button
+        class="md-icon-button"
+        aria-label="Settings"
+        ng-click="ctrl.showMenu($event, 'settings', 'toolbar')">
+        <md-icon md-svg-icon="img/icons/menu.svg"></md-icon>
+      </md-button>
+      <h2>Toolbar with grouped panels (Maximum open: 2)</h2>
+      <span flex></span>
+      <md-button
+        class="md-icon-button"
+        aria-label="Favorite"
+        ng-click="ctrl.showMenu($event, 'favorite', 'toolbar')">
+        <md-icon md-svg-icon="img/icons/favorite.svg"></md-icon>
+      </md-button>
+      <md-button
+        class="md-icon-button"
+        aria-label="More"
+        ng-click="ctrl.showMenu($event, 'more', 'toolbar')">
+        <md-icon md-svg-icon="img/icons/more_vert.svg"></md-icon>
+      </md-button>
+    </div>
+  </md-toolbar>
+
+  <md-content layout-padding>
+    <p>
+      Panels can be added to a group. Groups are used to configure specific
+      behaviors on multiple panels. To add a panel to a group, use the
+      <code>$mdPanel.newPanelGroup</code> method, or simply add a group name
+      to the configuration object passed into the <code>$mdPanel.create</code>
+      method.
+    </p>
+    <p>
+      Adding panels to groups prevents the panel API from storing a ridiculous
```

Comment:
reword - "rediculous amount of data" sounds too vague as does the added flexibility. This is your sales pitch for groups.